CONTRIBUTING.md

```diff
@@ -74,7 +74,7 @@ git remote add bsclifton git@github.com:bsclifton/brave-core.git
 git fetch bsclifton
 ```
 
-Once you're setup, there's a few tips we can suggest:
+Once you're set up, there's a few tips we can suggest:
```

Comment:
```suggestion
Once you're set up, there are a few tips we can suggest:
```

`tips` -> plural -> `there're` or `there are` (I chose the latter as it sounds a lot better).